document/src/vespa/document/fieldvalue/structfieldvalue.cpp

```diff
@@ -262,7 +261,7 @@ StructFieldValue::setFieldValue(const Field& field, FieldValue::UP value)
         _chunks.push_back(std::make_unique<SerializableArray>());
     }
 
-    _chunks.back().set(fieldId, std::move(serialized));
+    _chunks[0].set(fieldId, std::move(serialized));
```

Comment:
Since this changes the behavior from using chunk 1 to chunk 0 in the case of (legacy) 2 chunks, I presume we rather want to put it into the "header" chunk anyway?

Reply:
yes.